backend/app/controllers/spree/admin/refunds_controller.rb

```diff
@@ -8,6 +8,12 @@ class RefundsController < ResourceController
 
       rescue_from Spree::Core::GatewayError, with: :spree_core_gateway_error
 
+      def create
+        super.tap do
```

Comment:
This is probably not quite right. The return value of a controller action has no meaning, so we don't need to return the value from our superclass here.

However, the `save` performed in the `ResourceController`, our superclass, may or may not succeed. We don't want to perform a refund which failed to save.

We probably need to duplicate the logic from the ResourceController here, but inserting the `@refund.perform!` in the "success" path.